apps/src/StudioApp.js

```diff
@@ -809,7 +809,7 @@ StudioApp.prototype.renderShareFooter_ = function (container) {
       },
       {
         text: window.dashboard.i18n.t('footer.how_it_works'),
-        link: location.href + "/edit",
+        link: location.href + "/view",
```

Comment:
Ugh... so we seem to have at least three different share types with three different behaviors:

* For legacy shares /edit is valid but /view is 404 (regardless of login state?)
* For current artist/playlab shares
  * If you are the owner, /edit is valid and /view redirects to /edit
  * If you are a non-owner, /view is valid and /edit redirects to /view
  * If you are signed out, /view is valid and /edit redirects to /view
* For applab/gamelab shares
  * If you are the owner, /edit is valid and /view redirects to /edit
  * If you are a non-owner, /view is valid and /edit redirects to /view
  * If you are signed out, /view is valid and /edit **prompts you to sign in**

Semantically, it feels to me like "How it works" should always try to load /view, rather than /edit (which has the benefit of not hitting a login page for applab/gamelab shares) and perhaps legacy shares should redirect /view to /edit?  I think I prefer that to adding more conditional logic to the footer.

Actually it'd be even nicer to unify these behaviors.  The applab/gamelab behavior is my favorite - maybe with the addition that if you are signed in as a non-owner and go to /edit, we load /view but  add a callout to the Remix button.

That won't happen in this PR though.  Short-term fix options:

* Add a legacy share redirect from /view to /edit
* Make "How it works" go to /edit on legacy shares and /view otherwise
* Leave "How it works" pointing to /edit all the time and be okay that it hits a login page in some cases.